tests/e2e.sh

```diff
@@ -357,6 +355,27 @@ signer_modify_targets()
     git push --quiet origin $EVENT
 }
 
+signer_add_delegated_target()
```

Comment:
The test file really should get a file wide rename where "target" is changed to "artifact" where applicable: current state is confusing. But I didn't want to rename just the new methods and kept them in line with existing methods